src/test/java/io/aiven/kafka/connect/gcs/config/GcsSinkConfigTest.java

```diff
@@ -135,21 +137,34 @@ void correctlySupportDeprecatedYyyyUppercase() {
     @Test
     void requiredConfigurations() {
         final Map<String, String> properties = Map.of();
+
+        final var expectedErrorMessage =
+            "Missing required configuration \"gcs.bucket.name\" which has no default value.";
+
+        expectErrorMessageForConfigurationInConfigDefValidation(
```

Comment:
Sorry, I cannot get why we need an `assertEqual` on line 149, if this will also be checked here in `expectErrorMessageForConfigurationInConfigDefValidation`? Maybe I understand something wrong

Reply:
Ok. I think I got it. You check here static GCSSinkConfig method

Reply:
> But if a last check will work if this one is failing?

Definitely not, both need to pass for the test to pass.

> Isn't last one cover code for both?

It used to be thought of so. However, the point of this PR is to fix a bug that wasn't caught by the previous edition of the tests.